### DISCUSSION TITLE
吃饭还隔着骂娘，小米员工私自搞的xiaomikey一个收费65美金怎么不去骂，怎么不见小米贴公告出来制止这种行为，脸都没了

### DISCUSSION THREAD

**Discussion by pierrecashon** [MlgmXyysd/Xiaomi-BootLoader-Questionnaire#92] [General]
> 吃饭还隔着骂娘，小米员工私自搞的xiaomikey一个收费65美金怎么不去骂，怎么不见小米贴公告出来制止这种行为，脸都没了

你亲眼看到是小米内部员工搞的？？眼见为实，耳听为虚 这都不懂

_Originally posted by @Tomlls in https://github.com/MlgmXyysd/Xiaomi-BootLoader-Questionnaire/discussions/73#discussioncomment-8602749_

那你觉得是谁搞得，代码都摆出来了，事实摆在眼前了，还搁着自欺欺人？当猴当上瘾了？

**Comment by xbn2002** [comment]
最新：xiaomikey 大概率已寄
![image](https://github.com/MlgmXyysd/Xiaomi-BootLoader-Questionnaire/assets/27143040/66d02815-aea2-43c6-8ac5-24bf888fdf1f)

**Comment by pierrecashon** [comment]
好死，虽然我就是在他挂掉前一波解锁了，真真被耍的不轻，给这勾八玩意捞了不少钱

**Comment by Originalimoc** [comment]
对，企业大了，换人了，现在最担心的就是以后Gabe离开Valve后Steam系列会变成什么样子

**Comment by Tomlls** [comment]
> > 吃饭还隔着骂娘，小米员工私自搞的xiaomikey一个收费65美金怎么不去骂，怎么不见小米贴公告出来制止这种行为，脸都没了
> 
> 你亲眼看到是小米内部员工搞的？？眼见为实，耳听为虚 这都不懂
> 
> _Originally posted by @Tomlls in https://github.com/MlgmXyysd/Xiaomi-BootLoader-Questionnaire/discussions/73#discussioncomment-8602749_
> 
> 那你觉得是谁搞得，代码都摆出来了，事实摆在眼前了，还搁着自欺欺人？当猴当上瘾了？

吃你娘的饭，搞这种题库有屁用？搞笑

**Comment by Originalimoc** [comment]
突然发现，FRP/Reset这种都是可以绕的吗我原地哈哈，BL锁的唯一作用防盗=虚设
